### ISSUE TITLE
Clarification on Image Generation Steps and CLIP Score Differences in Table 1

### ISSUE THREAD

**Issue by zhang-lingyun** [rohitgandikota/erasing#31]
Hello, thank you for your work on this paper! I have a couple of questions regarding Table 1 in your paper:
1.	How many steps did you use to generate the images for computing the FID-30k scores?
2.	The CLIP scores for SLD and other methods seem to be around 0.15, whereas in the original SLD paper, they are typically around 0.75. Could you please clarify why the CLIP scores in Table 1 are on a different scale compared to the SLD paper?

Thank you for your time and clarification!

**Comment by rohitgandikota** [comment]
Hi - 

1.I do not recollect the number of steps we used for generation (I believe it is 50 steps) 
2. We were not sure why there is a discrepancy, we released our CLIP score evaluation script here (https://github.com/rohitgandikota/unified-concept-editing/blob/main/eval-scripts/mean_clip.py). feel free to take a look for reproduction. The follow up work UCE might also be a good reference for erasure https://unified.baulab.info
